### ISSUE TITLE
Code block tools: Running Tombi as a formatter over TOML fenced code blocks removes the content of the block

### ISSUE THREAD

**Issue by eread** [rvben/rumdl#527]
### rumdl version

0.1.47

### Installation method

mise

### Bug description

Code block tools (https://github.com/rvben/rumdl/blob/8c1798b13cf961c88a4c5d1136448f4f4391383d/docs/code-block-tools.md) can be used to run external tools over the contents of fenced code blocks to check they are valid, and reformat them if necessary.

However, when Tombi (https://tombi-toml.github.io/tombi/) is used as a formatter for TOML fenced code blocks, `rumdl fmt` removes the content of the fenced code block.

### Steps to reproduce

1. Create a Markdown file with this content:
   ````markdown
   ---
   title: Heading
   ---
   
   Here is a TOML fenced code block:
   
   ```toml
   key = "value"
   ```
   ````
1. Create a `.rumdl.toml` file with this content:
   ```toml
   [code-block-tools]
   enabled = true
   
   [code-block-tools.languages]
   toml = { lint = ["tombi"], format = ["tombi"] }
   ```
1. Run `rumdl fmt` on the Markdown file.

### Expected behavior

Given the scenario above, I'd expect no change to the Markdown file.

### Actual behavior

Given the scenario above, I end up with the following Markdown file:

````markdown
---
title: Heading
---

Here is a TOML fenced code block:

```toml

```
````

### Configuration

```toml
[code-block-tools]
enabled = true

[code-block-tools.languages]
toml = { lint = ["tombi"], format = ["tombi"] }
```

### Operating system

macOS

**Comment by rvben** [comment]
Fixed in v0.1.48. The code block tools formatter no longer erases content when a tool produces empty output. Thanks for reporting, @eread!
